app/src/main/java/dev/tuna/leechunbok/SplashViewModel.kt

```diff
@@ -0,0 +1,21 @@
+package dev.tuna.leechunbok
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.launch
+
+class SplashViewModel: ViewModel() {
+
+    private val _splashLoadingState = MutableStateFlow(true)
+    val splashLoadingState = _splashLoadingState.asStateFlow()
+
+    init {
+        viewModelScope.launch {
+            delay(2000)
```

Comment:
```suggestion
            delay(500)
```

영상을 위한 초로 보여서 우선 보기 위해서 500으로 제안해봅니다.